tests/browser/http/test_content_disposition.py

```diff
@@ -345,10 +355,10 @@ def test_attwithfnrawpctencaq(self):
 
         UA should offer to download the resource as "foo-%41.html".
         """
-        self._check_filename(r'attachment; filename="foo-%\41.html"',
-                             'foo-%41.html')
+        header_checker.check_filename(r'attachment; filename="foo-%\41.html"',
+                                      'foo-%41.html')
```

Comment:
Looks good to me (hanging indentation according to [PEP8](https://www.python.org/dev/peps/pep-0008/#indentation))